bls/signature.go

```diff
@@ -0,0 +1,243 @@
+//go:build relic
+
+package bls
+
+import (
+	"bytes"
+	"crypto/rand"
+	"encoding/hex"
+	"errors"
+	flowCrypto "github.com/onflow/flow-go/crypto"
+	"strings"
+)
+
+const SigningAlgorithm = flowCrypto.BLSBLS12381
+
+// TODO: what should the domainTag param be?
+var HashingAlgorithm = flowCrypto.NewExpandMsgXOFKMAC128("deso-protocol")
```

Comment:
It would be nice to have a high-level explanation up here of how the hasher fits into all this. Even if it's really simple like "The hashing algorithm will be used in aggregating and verifying signatures. It needs a distinct string because XYZ" would be nice.